.github/workflows/publish-documentation.yml

```diff
@@ -18,29 +18,40 @@ jobs:
     steps:
       - name: Checkout source
         uses: actions/checkout@v3
+        with:
+          path: main
+      - name: Checkout publish target
+        uses: actions/checkout@v3
+        with:
+          repository: graphql-dotnet/graphql-dotnet.github.io
+          token: ${{ secrets.ORG_PAT_TOKEN }}
+          path: published
       - name: Use Node.js ${{ env.NODE_VERSION }}
         uses: actions/setup-node@v3.3.0
         with:
           node-version: ${{ env.NODE_VERSION }}
       - name: yarn
         run: yarn
+        working-directory: main
       - name: yarn docs2
-        working-directory: docs2
+        working-directory: main/docs2
         run: yarn
       - name: yarn gatsby build
-        working-directory: docs2
+        working-directory: main/docs2
         run: yarn gatsby build
       - name: Publish docs artifacts
         uses: actions/upload-artifact@v3
         with:
           name: Documentation
-          path: docs2/public/**
+          path: main/docs2/public/**
           if-no-files-found: error
-      - name: Deploy documentation
-        uses: seanmiddleditch/gha-publish-to-git@v2
-        with:
-          repository: graphql-dotnet/graphql-dotnet.github.io
-          branch: master
-          github_token: '${{ secrets.GITHUB_TOKEN  }}'
-          github_pat: '${{ secrets.ORG_PAT_TOKEN }}'
-          source_folder: docs2/public
+      - name: Commit docs
+        shell: bash
+        run: |
+          rsync -a --quiet --delete --exclude ".git" "main/docs2/public/" "published/" || exit 1
```

Comment:
I don't use rsync, but I copied this from `seanmiddleditch/gha-publish-to-git`, the previously used action here.

Reply:
I see.